exercises/pov/src/POV.hs

```diff
@@ -3,7 +3,7 @@ module POV (fromPOV, tracePathBetween) where
 import Data.Tree (Tree)
 
 fromPOV :: Eq a => a -> Tree a -> Maybe (Tree a)
-fromPOV = error "You need to implement this function."
+fromPOV node tree = error "You need to implement this function."
 
 tracePathBetween :: Eq a => a -> a -> Tree a -> Maybe [a]
-tracePathBetween = error "You need to implement this function."
+tracePathBetween nodeA nodeB tree = error "You need to implement this function."
```

Comment:
do we want to make clear which node between the two given should be first in the path?